src/overlays/actors/ovl_Obj_Um/z_obj_um.h

```diff
@@ -9,6 +9,17 @@ struct ObjUm;
 
 typedef void (*ObjUmActionFunc)(struct ObjUm*, PlayState*);
 
+#define MILK_POTS_COUNT 3
+
+typedef enum ObjUmAnimimations {
+    /* -1 */ OBJ_UM_ANIM_MINUS_1 = -1,
```

Comment:
I've seen this used for just no animation in other actors, would `NONE` fit here?

Reply:
No, it is a hacky way to trigger the LOOK_BACK animation